src/MongoDB.Bson/IO/IBsonReaderExtensions.cs

```diff
@@ -91,8 +91,8 @@ public static BsonBinaryData ReadBinaryData(this IBsonReader reader, string name
         /// <summary>
         /// Reads a BSON binary data element from the reader temporarily setting the GuidRepresentation to Unspecified.
         /// </summary>
-        /// <param name="reader"></param>
-        /// <returns></returns>
+        /// <param name="reader">The reader.</param>
+        /// <returns>A BsonBinaryData.</returns>
```

Comment:
Actually we have more empty parameter descriptions in the code. You can find them via searching `><`. Should we also fix them?

Reply:
It is surprising that Sandcastle did not issue a warning.

In the past we have been content to just fix Sandcastle warnings. It's unfortunate if that's not enough.

I will search for and fix these.

Reply:
I looked for missing <param> and <returns> doc comments (ones that Sandcastle for some reason was not warning about) and added them.

I am re-requesting a review to proof read the new doc comments.